app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -62,6 +62,17 @@ $(function() {
           width: 'resolve',
           allowClear: allowClear
         });
+
+        var $parent = $(el).parent(), $label = $parent.find('label');
+
+        if ($label.length) {
+          var $input = $parent.find('#' + $label.attr('for'))
```

Comment:
Unexpected var, use let or const instead                           no-var<br>All 'var' declarations must be at the top of the function scope    vars-on-top<br>Unexpected string concatenation                                    prefer-template<br>Missing semicolon                                                  semi